generators/app/templates/index.html

```diff
@@ -107,5 +107,8 @@ <h3 class="text-muted"><%= appName %></h3>
       r.parentNode.insertBefore(e,r)}(window,document,'script','ga'));
       ga('create','UA-XXXXX-X');ga('send','pageview');
     </script>
+<% if (includeRequireJS) { -%>
+    <script data-main="scripts/main" src="bower_components/requirejs/require.js"></script>
+<% } -%>
```

Comment:
simple and clever solution to bypass the usemin block generated by `htmlWiring.appendScript` :+1: nice job!